openapi/convert_openapi_to_swagger.sh

```diff
@@ -0,0 +1 @@
+api-spec-converter  --from=openapi_3 --to=swagger_2 ./openapi.json > swagger.json
```

Comment:
add comment into script , that specify why we use this conversion and link to install locally https://www.npmjs.com/package/api-spec-converter , useful to other developers